web/packages/teleterm/src/agentCleanupDaemon/agentCleanupDaemon.test.ts

```diff
@@ -80,6 +80,13 @@ describe('agentCleanupDaemon', () => {
       expect(isRunning(pids['agentCleanupDaemon'])).toBe(true);
 
       // Verify that killing the parent results in the eventual termination of both children.
+      //
+      // Note that when the parent is killed, the child processes become orphans (https://en.wikipedia.org/wiki/Orphan_process),
+      // and when orphans are killed, they become zombies (https://en.wikipedia.org/wiki/Zombie_process).
+      // In typical UNIX environments, zombies exist only momentarily and are cleaned up by the init process (https://en.wikipedia.org/wiki/Init),
+      // however in Docker there is no init process by default (https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/),
+      // so zombie processes end up sticking around until the container is stopped. Hence in order for this test to pass in a `docker run`,
+      // we need to add the `--init` flag (https://docs.docker.com/engine/reference/run/#specify-an-init-process).
```

Comment:
Where is said `--init` flag?

Reply:
It's used here:

https://github.com/gravitational/teleport/pull/35401/files#diff-6c099f0fc9d8f8a6677998cdb8f0f619e662a887bf91adbe15d7a2dc5af57ebbR22

Reply:
Gotcha. Any reason why the large comment explaining why the flag is necessary is so far away from where the flag is used? Would it make more sense to move this reasoning over there?

Reply:
I understood the need for that flag in close relation to the behavior of this specific test, so wrote it here in order to codify that. IMO I would need to link to this test in order to properly document why `init` was added over in `unit-tests-ui.yaml`, so I figured I may as well leave the explanation where it is and [just link to explanation + test](https://github.com/gravitational/teleport/pull/35401/files#diff-6c099f0fc9d8f8a6677998cdb8f0f619e662a887bf91adbe15d7a2dc5af57ebbR20-R21).